### DISCUSSION TITLE
Changing the repository name

### DISCUSSION THREAD

**Discussion by kennedyyung** [community/community#60752] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys, anyone know what to do to change the repository name so it changes the website link. I changed it but realized its probably more complicated so I changed the repo back but now my website is not deploying the same way before and all the design is gone. anything helps! Thank you

**Comment by queenofcorgis** [comment]
Hi there @kennedyyung, if you plan to rename a repository that has a GitHub Pages site, we recommend using a custom domain for your site. This ensures that the site's URL isn't impacted by renaming the repository. For more information, see "[About custom domains and GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages).
